File: client/src/app/shared/models/investmentRecDepot.ts
export interface IInvestmentRecDepot {
    id: number;
    investmentInitId: number;
    depotCode: string;
    depotName: string;
    employeeId: number;
}
 
export class InvestmentRecDepot implements IInvestmentRecDepot {
    id: number=0;
    investmentInitId: number;
    depotCode: string;
    depotName: string;
    employeeId: number;
}

File: Core/Entities/InvestmentRecDepot.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace Core.Entities
{
    public class InvestmentRecDepot : BaseEntity
    {
        public int? InvestmentInitId { get; set; }
        [ForeignKey("InvestmentInitId")]
        public InvestmentInit InvestmentInit { get; set; }
        public int EmployeeId { get; set; }
        public Employee Employee { get; set; }
        public string DepotCode { get; set; }
        public string DepotName { get; set; }
    }
}
